src/tasks/withdraw/traded_tokens.ts

```diff
@@ -1,8 +1,37 @@
+import axios from "axios";
 import { Contract } from "ethers";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { BUY_ETH_ADDRESS } from "../../ts";
 
+export const BUFFER_TRADABLE_TOKENS: Promise<string[]> = axios
```

Comment:
nit: I would personally have made this a function:
```ts
export async function fetchTokensFromTokenList(tokenListUrl: string): Promise<string[]> {
    // ...
}
```

Since we only use it once, there isn't much benefit to caching the result.